pkg/controller/applicationmonitoring/templateHelper.go

```diff
@@ -208,7 +213,7 @@ func (h *TemplateHelper) loadTemplate(name string) ([]byte, error) {
 	parser := template.New("application-monitoring")
 	parser.Funcs(template.FuncMap{
 		"JoinQuote": joinQuote,
-	})
+	}).Funcs(sprig.TxtFuncMap())
```

Comment:
this does not seem to be used in the templates

Reply:
It's used in both the `prometheus.yml` and `alertmanager.yml` files to indent: [alertmanager.yml](https://github.com/integr8ly/application-monitoring-operator/pull/152/files#diff-4be66ac58044fd8db91c27c657ba4c2be73e29eed2c70721ea95842f2e2769ebR54)